source/geometries/NextDemoTrackingPlane.cc

```diff
@@ -3,8 +3,8 @@
 //
 // Tracking plane of the Demo++ geometry.
 // It implements 2 different TP implementations:
-// "original": the one used from the very beginning with passing-holes masks
-// "type 1": the one used in the so-called Run7, with coated masks and membranes.
+// "run5": the one used from the very beginning with passing-holes masks
```

Comment:
I would remove "from the very beginning" or replace it with "at the beginning", since it is applies only to that run.

Reply:
Done

Reply:
I don't see the change...